### PR TITLE
e2e: pin cypress image

### PR DESCRIPTION
**Pull request by BacLuc**
Even to digest, that we control it when the image changes-

In a future PR:
Get renovate to update this image and the version in the package.json to be updated synchronously, but without auto merge.

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -140,6 +140,12 @@
         "postgres"
       ],
       "allowedVersions": "15-alpine"
+    },
+    {
+      "matchDepNames": [
+        ".*cypress.*"
```
Comment (simfeld) [inline renovate.json]: are we sure this only targets the Cypress image tag but not npm dependencies like `eslint-plugin-cypress`?

Comment (BacLuc) [reply renovate.json]: Jup, done.
Result can be seen here: https://github.com/BacLuc/ecamp3/commit/7f5b210092fbb965496142de2ffa81b547b2e78a

The auto update of the cypress docker image does not work yet.
-> planned to do in another PR
(for now we rather don't want cypress to be updated)

**Review by simfeld — APPROVED** [review]

**Review by manuelmeister — COMMENTED** [review]
Can't we just use the index digest, instead of linking to a specific architecture?
Bildschirmfoto 2024-08-16 um 15 48 49

https://hub.docker.com/layers/cypress/included/13.13.0/images/sha256-f9733a2cadc3aa270e40f8ce1158a23cb99703476a9db7154b4ecc51ba02bd5c

**Comment by BacLuc** [comment]
> Can't we just use the index digest, instead of linking to a specific architecture? Bildschirmfoto 2024-08-16 um 15 48 49
> 
> https://hub.docker.com/layers/cypress/included/13.13.0/images/sha256-f9733a2cadc3aa270e40f8ce1158a23cb99703476a9db7154b4ecc51ba02bd5c

It seems such a mechanism is possible.
```
lucius@lucius-ThinkPad-P17-Gen-1:~/Downloads$ docker buildx imagetools inspect cypress/included@sha256:f9733a2cadc3aa270e40f8ce1158a23cb99703476a9db7154b4ecc51ba02bd5c
Name:      docker.io/cypress/included@sha256:f9733a2cadc3aa270e40f8ce1158a23cb99703476a9db7154b4ecc51ba02bd5c
MediaType: application/vnd.docker.distribution.manifest.list.v2+json
Digest:    sha256:f9733a2cadc3aa270e40f8ce1158a23cb99703476a9db7154b4ecc51ba02bd5c
           
Manifests: 
  Name:      docker.io/cypress/included@sha256:f8302f21218f3bb485c842d6f0581b17201ae8ff60c11fc09cce005ee660e1fd
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/arm64
             
  Name:      docker.io/cypress/included@sha256:542b11720d448df6d7b12f85015f5adcb2b7f4618af4e77cd28b62766c0d3491
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/amd64
```

i suggest you test if cypress/included@sha256:f9733a2cadc3aa270e40f8ce1158a23cb99703476a9db7154b4ecc51ba02bd5c works on macos.
If yes, we can use this mechanism

**Comment by manuelmeister** [comment]
@BacLuc it works:
```
image: cypress/included:cypress-13.13.0-node-20.15.1-chrome-126.0.6478.114-1-ff-128.0-edge-126.0.2592.61-1@sha256:f9733a2cadc3aa270e40f8ce1158a23cb99703476a9db7154b4ecc51ba02bd5c
```

I would include the "browser description"

**Review by manuelmeister — APPROVED** [review]

**Pull request merged**